### PR TITLE
Fix SetupState breakage with upcoming pytest 6.3

### PR DESCRIPTION
**Pull request by bluetech**
pytest 6.3 makes some changes to the SetupState class, adjust to that.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -24,6 +24,8 @@ deps =
     pytest54: pytest==5.4.*
     pytest60: pytest==6.0.*
     pytest61: pytest==6.1.*
+    pytest62: pytest==6.2.*
+    pytestmaster: git+https://github.com/pytest-dev/pytest.git@master#egg=pytest
```
Comment (icemac) [inline tox.ini:28]: I like testing against `master`.
This change should also be applied to `.github/workflows/test.yml` so CI runs them, too.

Comment (bluetech) [reply tox.ini:28]: Sure, added. Using master might of course cause breakage from the pytest side, but hopefully not.

**Review by icemac — APPROVED** [review]
Thank you for this PR and improving the test setup for this plug-in.
I like it.

**Review by sallner — APPROVED** [review]
I like the changes, too.

**Comment by icemac** [comment]
Thank you for this PR and the review. 😃

**Pull request merged**